cdragontoolbox/wad.py

```diff
@@ -120,7 +121,13 @@ def guess_extension(data):
 
 class Wad:
     """
-    WAD archive
+    A WAD archive is a file that contains other files. It has a header that describes the format. There are multiple
```

Comment:
Multi-line docstrings should start with a single-line summary of the documented object.
See: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Fixed